data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Ik stel voor een nieuw Pull-request te maken van de huidige wijzigingen in https://github.com/OpenRCT2/Localisation, en voor deze regel een apart PR te maken. Op die manier zitten de vertalingen eindelijk op de goede plek, en kunnen de wijzigingen waarover iedereen het eens is tenminste gemerged worden.